src/API/OnboardingTasks.php

```diff
@@ -114,6 +114,12 @@ public function register_routes() {
 			$this->namespace,
 			'/' . $this->rest_base,
 			array(
+				'args'   => array(
+					'only_viewable' => array(
```

Comment:
I like "viewable" but I'm not a huge fan of the `only_` prefix.  Can we prefix with a verb?  `filter_` possibly?

Reply:
I mentioned in [this other earlier comment](https://github.com/woocommerce/woocommerce-admin/pull/7718/files#r718034370), I'm good with changing this, but `filter_viewable` seems ambiguous to me. Would `filter_only_viewable` work for you?

Reply:
This seems to be moot now that I removed the endpoint parameter ✅